### PR TITLE
Map the Debris

### PR DESCRIPTION
**Pull request by breyana**

### CODE REVIEW

File: algorithms/map_the_debris.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+The values should be rounded to the nearest whole number. The body being orbited
+ is Earth.
+
+The radius of the earth is 6367.4447 kilometers, and the GM value of earth is
+398600.4418 km3s-2.
+
+*/
+
+function orbitalPeriod(arr) {
+  var GM = 398600.4418;
```
Comment (punitrathore) [inline algorithms/map_the_debris.js]: variables which are constants should be declared globally, and the name should be capitalized. So in your code `GM` &  `EARTHRADIUIS` are global constants

File: algorithms/map_the_debris.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    objectSphere.name = next.name;
+    objectSphere.orbitalPeriod = Math.round(constant*Math.PI*Math.pow( Math.pow( earthRadius + next.avgAlt , 3 ) / GM , 1/2 ));
+    console.log(objectSphere);
+    arrayofObjects.push(objectSphere);
+  }, []);
+
+  return arrayofObjects;
+}
+
+orbitalPeriod([{name: "iss", avgAlt: 413.6}, {name: "hubble", avgAlt: 556.7},
```
Comment (punitrathore) [inline algorithms/map_the_debris.js:35]: Can you write a few tests for this function?

File: algorithms/map_the_debris.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+*/
+
+function orbitalPeriod(arr) {
+  var GM = 398600.4418;
+  var earthRadius = 6367.4447;
+  var constant = 2;
+  var arrayofObjects = [];
+
+  var finalObject = arr.reduce(function(accumulator, next){
```
Comment (punitrathore) [inline algorithms/map_the_debris.js]: Love that you are using `reduce`. Can you name the variable `next` to something that indicates what the value of it is?

**Comment by breyana** [comment]
@punitrathore added in some of your suggestions -- we'll be making a new branch with creating mocha tests for these functions

**Pull request merged**